default.nix

```diff
@@ -72,7 +72,8 @@ in
           pyrsistent
           configparser
           eliot
-          foolscap >= 21.7.0
+          foolscap
+          collections-extended >= 2.0.2
```

Comment:
Tahoe-LAFS master@HEAD now has an environment marker in the collections-extended requirement declaration.